### ISSUE TITLE
Minification bug on Windows

### ISSUE THREAD

**Issue by borekb** [pingyhq/pingy-cli#34]
`pingy export` yielded this error:

```
{ Error: ENOENT: no such file or directory, open 'c:\example\$stdin'
    at Error (native)
  errno: -4058,
  code: 'ENOENT',
  syscall: 'open',
  path: 'c:\\example\\$stdin' }
\ Compiling main.less
```

When I went to [cli-run.js#L77](https://github.com/pingyhq/pingy-cli/blob/1a19c238bee20b43b8442e3b54c56a2a8b86d2f9/packages/cli/cli-run.js#L77) and removed `minify: true`, all was fine.

**Comment by borekb** [comment]
Related to this, I was looking whether minification can be disabled in `.pingy.json` but it doesn't seem so. Would be useful, IMO.

**Comment by davej** [comment]
Yeah. These are both bugs. On holiday at the moment but they are high priority when I'm back.

**Comment by davej** [comment]
Thanks for reporting :+1:

**Comment by davej** [comment]
Options in `.pingy.json` are now respected as of `v0.6.8`: https://github.com/pingyhq/pingy-cli/releases/tag/%40pingy%2Fcli%400.6.8

**Comment by borekb** [comment]
Works great, thank you!
